.gitignore

```diff
@@ -0,0 +1,10 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
I added a .gitignore file as well so we don't congest the master repo with cache files or IDE settings.